### PR TITLE
[ar][MRG] Add Arabic translation of tensorflow/tutorials/keras/classification

### PR DESCRIPTION
**Pull request by mohamed-ali**
This is a translation of the tensorflow/tutorials/keras/classification notebook into Arabic.

Here is the original link of the tutorial: https://www.tensorflow.org/tutorials/keras/classification

### CODE REVIEW

File: site/ar/tutorials/keras/classification.ipynb [hunk 1]
```diff
@@ -0,0 +1,1437 @@
...
+        "\n",
+        "قبل أن يصبح النموذج جاهزًا للتدريب ، يحتاج إلى بعض الإعدادات الإضافية \n",
+        "و التّي تتمّ إضافتها \n",
+        "في خطوة تجميع النّموذج\n",
+        "(model compiling)\n",
+        "و التّي تقوم بها الدّالة \n",
+        "`model.compile`.\n",
+        "و هذه الإعدادات هي:\n",
+        "\n",
+        "* *وظيفة الخسارة*\n",
```
Comment (baheytharwat) [inline site/ar/tutorials/keras/classification.ipynb]: دالة الخسارة

File: site/ar/tutorials/keras/classification.ipynb [hunk 2]
```diff
@@ -0,0 +1,1437 @@
...
+        "(Loss function) - \n",
+        " تقيس مدى دقّة النموذج أثناء التدريب. هدفك هو أن تبنِيَ نموذجا يقوم بتقليل قيمة هذه الوظفية في كُلّ خطوة من عمليّة التدريب حتى تقوم \"بتوجيه\" النموذج في الاتجاه الصحيح. \n",
```
Comment (baheytharwat) [inline site/ar/tutorials/keras/classification.ipynb]: دالة أو وظيفة بدلا من وظفية

File: site/ar/tutorials/keras/classification.ipynb [hunk 3]
```diff
@@ -0,0 +1,1437 @@
...
+        "Fashion MNIST\n",
+        "لتكون بديلا مباشرا لمجموعة البيانات الكلاسيكية \n",
+        "[MNIST](http://yann.lecun.com/exdb/mnist/)\n",
+        "و التّي تعتبر المثال الأوّل ، أو مثال\n",
+        "\"مرحبا يا عالم\"\n",
+        "(\"Hello, word\")\n",
+        "لبرامج تعلّم الآلة في مجال رؤية الكمبيوتر. \n",
+        "و تحتوى مجموعة بيانات \n",
+        "MNIST\n",
+        "على صور للأعداد من 0 إلى 9 مكتوبة بخط اليد من قِبَلِ العديد من الأشخاص حيث أنّ تنسيق الصور فيها ممثال لتنسيق الصور الموجودة في \n",
```
Comment (baheytharwat) [inline site/ar/tutorials/keras/classification.ipynb]: مماثل بدلا من ممثال

File: site/ar/tutorials/keras/classification.ipynb [hunk 4]
```diff
@@ -0,0 +1,1437 @@
...
+        "colab_type": "text",
+        "id": "FbVhjPpzn6BM"
+      },
+      "source": [
+        "<div dir=\"rtl\">\n",
+        "\n",
+        "يقوم هذا الدّليل بتدريب نموذج شبكة عصبيّة لتصنيف الملابس، مثل الأحذية الرياضية و القمصان. \n",
+        "لابأس إن لم تفهم كلّ التّفاصيل، \n",
+        "فهذا درس تفاعليّ ذو نسق سريع\n",
+        "يعطي نظرة عامّة عن كيفية إستعمال  \n",
```
[the remark this answers is not in the document]
Comment (mohamed-ali) [reply site/ar/tutorials/keras/classification.ipynb]: I'll push a fix. Thanks

**Comment by mohamed-ali** [comment]
@baheytharwat @tayciryahmed Thanks for reviewing. 

I addressed your comments and did some further fixes when relevant.

File: site/ar/tutorials/keras/classification.ipynb [hunk 5]
```diff
@@ -0,0 +1,1439 @@
...
+        "</table>"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "FbVhjPpzn6BM"
+      },
+      "source": [
```
Comment (lamberta) [inline site/ar/tutorials/keras/classification.ipynb]: we need to add the community translation disclaimer to the top of notebooks.
this is in the README in this PR: https://github.com/tensorflow/docs-l10n/pull/108 ---but still is in review. Might want to break the README into a separate PR to get it in

Comment (mohamed-ali) [reply site/ar/tutorials/keras/classification.ipynb]: @lamberta  are you referring to this text: https://github.com/tensorflow/docs-l10n#content ?

**Comment by mohamed-ali** [comment]
@lamberta is this the disclaimer you were referring to? 

> Our TensorFlow community has translated these documents. Because community translations are best-effort, there is no guarantee that this is an accurate and up-to-date reflection of the official English documentation. If you have suggestions to improve this translation, please send a pull request to the tensorflow/docs GitHub repository. To volunteer to write or review community translations, contact the docs@tensorflow.org list.

**Comment by lamberta** [comment]
Yes, that's right. Just prepend it with `Note: ...` (non-translated since this is a callout style)
Here's a Korean notebook example: https://colab.research.google.com/github/tensorflow/docs-l10n/blob/master/site/ko/tutorials/quickstart/beginner.ipynb#scrollTo=_dt4BrWOPl-C

File: site/ar/tutorials/keras/classification.ipynb (hunk 5 above)
Comment (tayciryahmed) [reply site/ar/tutorials/keras/classification.ipynb]: @mohamed-ali @baheytharwat I suggest this translation and will include it in the notebook of  my currently pending [PR](https://github.com/tensorflow/docs-l10n/pull/142). 

<div dir='rtl'>
 قامت مجموعة المتطوعين  لTensorflow  بترجمة هذه المستندات. نظرًا لأن ترجمات المجموعة تعتمد على قاعدة أقصى الجهد  ، فليس هناك ما يضمن أنها انعكاس دقيق وحديث للوثائق الإنجليزية الرسمية. إذا كانت لديك اقتراحات حول كيفية تحسين هذه الترجمة ، يرجى إرسال "Pull request" إلى مشروع tensorflow/docs . للتطوع أو مراجعة ترجمات المجموعة يرجى ارسال ايميل إلى  docs@tensorflow.org.

</div>

Comment (mohamed-ali) [reply site/ar/tutorials/keras/classification.ipynb]: @tayciryahmed thanks for your suggestion. 

I suggest few edits, which result in this disclaimer 

<div dir="rtl">

 قامت مجموعة المتطوعين في مشروع Tensorflow بترجمة هذا المحتوى. نظرًا لأن هذه الترجمات تعتمد على قاعدة أقصى الجهد (best-effort) ،  فلا نضمن أنها انعكاس دقيق وحديث للمحتوى الرسمي باللغة الإنجليزية. إذا كانت لديك اقتراحات لتحسين هذه الترجمة ، يرجى إرسال "Pull request" إلى مشروع tensorflow/docs . للتطوع أو مراجعة ترجمات المجموعة يرجى ارسال ايميل إلى docs@tensorflow.org.

</div>

Comment (tayciryahmed) [reply site/ar/tutorials/keras/classification.ipynb]: Agreed.

**Comment by mohamed-ali** [comment]
@lamberta I added the disclaimer, could you review again? Thanks.

File: site/ar/tutorials/keras/classification.ipynb [hunk 6]
```diff
@@ -0,0 +1,1442 @@
...
+        "#\n",
+        "# Unless required by applicable law or agreed to in writing, software\n",
+        "# distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+        "# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+        "# See the License for the specific language governing permissions and\n",
+        "# limitations under the License."
+      ]
+    },
+    {
+      "cell_type": "code",
```
[the remark this answers is not in the document]
Comment (mohamed-ali) [reply site/ar/tutorials/keras/classification.ipynb]: yes I am running nbfmt.py before all my commits.

**Comment by mohamed-ali** [comment]
@lamberta  I added the resources links.

**Comment by mohamed-ali** [comment]
@lamberta the tutorials is still in english when choosing the Arabic language in Tensorflow's website: https://www.tensorflow.org/tutorials/keras/classification?hl=ar. 

Was the deployment pipeline temporary disabled for some reason?

**Comment by yashk2810** [comment]
I'll fix it. The pipeline wasn't automated. Thanks for reporting!

**Comment by mohamed-ali** [comment]
@yashk2810 @lamberta this tutorial is still showing the english version when selecting the Arabic language: https://www.tensorflow.org/tutorials/keras/classification?hl=ar

**Comment by yashk2810** [comment]
Yup, I am working on it 😄

**Comment by mohamed-ali** [comment]
@yashk2810 Thank you! Sorry for insisting, I want to use these tutorials as a leverage to expand  the community :)

**Comment by yashk2810** [comment]
The tutorial should be up in 30 minutes or so.

The automation will update the `ar` version of the site every Friday morning PST.

**Comment by yashk2810** [comment]
This is now live: https://www.tensorflow.org/tutorials/keras/classification?hl=ar

**Comment by mohamed-ali** [comment]
@yashk2810  Great work, thanks for your effort! 

There seem to be an issue however for this tutorial: The first part is rendering correctly, but from this section https://www.tensorflow.org/tutorials/keras/classification?hl=ar, the rendering is broken. 

I went through the tutorial again to see if there is a problem, but can't spot any issue for now.

The other tutorial: https://www.tensorflow.org/tutorials/text/nmt_with_attention?hl=ar is rendering correctly. 

Thanks again.

**Comment by lamberta** [comment]
@yashk2810  Looks like the formatting breaks around this cell: https://colab.research.google.com/github/tensorflow/docs-l10n/blob/master/site/ar/tutorials/keras/classification.ipynb#scrollTo=TRFYHB2mCaWb&line=1&uniqifier=1

There's a runaway ` ```python len(train_labels) ``` `

**Comment by yashk2810** [comment]
The div wasn't closed: fix here https://github.com/tensorflow/docs-l10n/pull/163
